src/common/Header/index.js

```diff
@@ -1,8 +1,13 @@
-import React from "react";
-import { StyledHeader } from "./styled";
+import styled from "styled-components";
 
-const Header = (props) => {
-  return <StyledHeader>{props.children}</StyledHeader>;
-};
+export default styled.h1`
+  margin: 56px 0 24px 0;
+  font-weight: 600;
+  font-size: 36px;
+  line-height: 1.2;
 
-export default Header;
+  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
+    margin: 24px 0 12px 0;
+    font-size: 18px;
+  } ;
+`;
```

Comment:
Nie miałem pojęcia że można tak ten komponent zrobić, ale jeżeli działa to super pomysł, bardzo przejrzyście jest